ibr/binToJs.js

```diff
@@ -7,6 +7,9 @@
 import {OrbitControls} from
   './node_modules/three/examples/jsm/controls/OrbitControls.js';
 
+// global variable to keep track of structure's index for height adjustment
+let structureIndex = 0;
```

Comment:
global variables are generally bad, is there another way to implement this? I only see structureIndex referenced in extractSingleStructureData, can we keep it in there?